api/src/main/resources/moduleApplicationContext.xml

```diff
@@ -18,6 +18,6 @@
   		    http://www.springframework.org/schema/util/spring-util-3.0.xsd">
 
 	<!-- Add here beans related to the API context -->
-
+	<context:component-scan base-package="org.openmrs.module.referenceapplication.builtinreports" />
```

Comment:
Is this necessary? Could you please try removing it? If it is needed, then it should look for components in the whole module i.e. org.openmrs.module.referenceapplication.